app/templates/components/orders/order-summary.hbs

```diff
@@ -39,63 +39,94 @@
     <h3 class="weight-400">{{t 'Order Summary'}}</h3>
   </div>
   {{#if (or (eq session.currentRouteName 'orders.new') (eq session.currentRouteName 'orders.view') (eq session.currentRouteName 'orders.pending'))}}
-    <table class="ui very basic tablet stackable table order-summary center aligned">
-      <thead>
-        <tr>
-          <th class="four wide">{{t 'Ticket Type'}}</th>
-          <th class="four wide">{{t 'Price'}}</th>
-          <th class="ui aligned two wide">{{t 'Discount'}}</th>
-          <th class="one wide">{{t 'Quantity'}}</th>
-          <th class="ui aligned two wide">{{t 'Subtotal'}}</th>
-          {{#if event.tax}}
-            <th class="four wide">{{t 'Tax'}}</th>
-          {{/if}}
-        </tr>
-      </thead>
-      <tbody>
-        {{#each tickets as |ticket|}}
+    {{#if (eq data.amount 0)}}
```

Comment:
what if the amount is 0 BECAUSE of a discount code.

Reply:
Actually the second screenshot was for test purpose , How would it look and for Discount code with net amount zero , A column for `Discount` and `Ticket Price` would be shown without `Subtotal`